snapshots/output/react/src/LoaderInput.tsx

```diff
@@ -26,9 +26,14 @@
 //                                ^^^^^^^^^^^^^^^^^ reference @types/react 17.0.0 `index.d.ts`/React/FunctionComponent#
 //                                                  ^^^^^ reference react-example 1.0.0 src/`LoaderInput.tsx`/Props#
     loading,
-//  ^^^^^^^ reference local 3
+//  ^^^^^^^ definition local 3
+//  documentation ```ts\n(parameter) loading: boolean\n```
+//  ^^^^^^^ reference react-example 1.0.0 src/`LoaderInput.tsx`/Props#loading.
     children,
-//  ^^^^^^^^ reference local 4
+//  ^^^^^^^^ definition local 4
+//  documentation ```ts\n(parameter) children: ReactNode\n```
+//  ^^^^^^^^ reference react-example 1.0.0 src/`LoaderInput.tsx`/Props#children.
+//  ^^^^^^^^ reference local 7
```

Comment:
Why is there an extra `reference local` here?

Reply:
Good catch, opened https://github.com/sourcegraph/scip-typescript/issues/188